src/main/java/BaseBall.java

```diff
@@ -0,0 +1,174 @@
+import java.util.Scanner;
+
+public class BaseBall {
+    public static void main(String[] args) {
+
+        new TDDClass().run();
+
+        Scanner scanner = new Scanner(System.in);
+        int size;
+        while (true) {
+            System.out.println("몇 자리로 하시겠습니까?");
+            size = Integer.parseInt(scanner.nextLine());
+            if (size <= 10) {
+                break;
+            } else {
+                System.out.println("10자리 이하로만 가능합니다.");
+            }
+        }
+
+        String rand = new RandomNumber().getNumber(size);
```

Comment:
변수명을 축약해서 쓰는 것은 별로 좋지 않습니다.
그리고 아래와 같이 변수명에 의미를 담아주는게 이 변수가 어떤 역할을 가지고 있는지 파악하는데 도움이 됩니다.

```suggestion
        String answers = new RandomNumber().getNumber(size);
```